packages/p2p-v2/src/utils/time.ts

```diff
@@ -148,3 +148,17 @@ export const formatMilliseconds = (miliseconds: moment.MomentInput, strFormat: s
     }
     return moment.utc(miliseconds).format(strFormat);
 };
+
+/**
+ * Gets the date string after the given number of hours.
+ * @param {Number} initialEpoch - The initial epoch time.
+ * @param {Number} hours - The number of hours to add.
+ * @returns {String} The date string after the given number of hours.
+ */
+export const getDateAfterHours = (initialEpoch: number, hours: number): string => {
```

Comment:
Maybe we can give this to deriv-com/utils
What do you think @niloofar-deriv

Reply:
we could, we just need to cross check with other squads if they want to display the date with hours like this `21 Mar 2024, 01:10` for example